templates/homepage.julius

```diff
@@ -22,6 +22,7 @@ $(function() {
       success: function (data) {
         var newNode = $("<li></li>");
         newNode.text(data.message);
+        console.log(data);
         $("##{rawJS commentListId}").append(newNode);
```

Comment:
this is actually a feature not a bug ;) I have added this [text](https://github.com/yesodweb/yesod-scaffold/pull/135/files#diff-a8900c26af8ddefc60773f874e4506c4R124), so devs can see the response object.